packages/fluent/scss/pager/_variables.scss

```diff
@@ -0,0 +1,191 @@
+@use "../core/color-system" as *;
+@use "../core/_variables.scss" as *;
+@use "../button/_variables.scss" as *;
+@use "../list/_variables.scss" as *;
+
+/// Horizontal padding of the pager.
+/// @group pager
+$kendo-pager-padding-x: map-get( $kendo-spacing, 2 ) !default;
+/// Vertical padding of the pager.
+/// @group pager
+$kendo-pager-padding-y: map-get( $kendo-spacing, 2 ) !default;
+/// Border width of the pager.
+/// @group pager
+$kendo-pager-border-width: 1px !default;
+
+/// Font family of the pager.
+/// @group pager
+$kendo-pager-font-family: var( --kendo-font-family, normal ) !default;
+/// Font size of the pager.
+/// @group pager
+$kendo-pager-font-size: var( --kendo-font-size, inherit ) !default;
+/// Line height of the pager.
+/// @group pager
+$kendo-pager-line-height: var( --kendo-line-height, normal ) !default;
+
+/// Background color of the pager.
+/// @group pager
+$kendo-pager-bg: var( --kendo-component-bg, transparent ) !default;
+/// Text color of the pager.
+/// @group pager
+$kendo-pager-text: var( --kendo-component-text, inherit ) !default;
+/// Border color of the pager.
+/// @group pager
+$kendo-pager-border: var( --kendo-component-border, inherit ) !default;
+
+/// Background color of the focused pager.
+/// @group pager
+$kendo-pager-focus-bg: $kendo-pager-bg !default;
+/// Text color of the focused pager.
+/// @group pager
+$kendo-pager-focus-text: $kendo-pager-text !default;
+/// Border color of the focused pager.
+/// @group pager
+$kendo-pager-focus-border: get-theme-color-var( neutral-130 ) !default;
+
+/// Spacing between the pager sections.
+/// @group pager
+$kendo-pager-section-spacing: 1em !default;
+
+/// Horizontal padding of the pager items.
+/// @group pager
+$kendo-pager-item-padding-x: $kendo-button-padding-y-md !default;
```

Comment:
add css variable:

var( --kendo-button-padding-y-md, #{$kendo-button-padding-y-md});

The same goes for the rest of the SASS variables - kendo-pager-bg, kendo-pager-text, etc.

In this way we're making sure that if CSS vars are used for customization, it'll be applied to the pager as well